Server/MirObjects/PlayerObject.cs

```diff
@@ -1487,6 +1496,84 @@ public void AddObserver(MirConnection observer)
             if (observer.Player != null)
                 observer.Player.StopGame(24);
         }
+
+        protected virtual void GetAllItemInfo(MirConnection c)
+        {
+            // Send all ItemInfo to client for item tooltips
+            foreach (var itemInfo in Envir.ItemInfoList)
```

Comment:
Sending the entire item database to the client each time as single packets will have a big performance hit no doubt. You should really just be sending the item once its been requested by the client.